website/docs/tutorials/07-testing/03-fuzzing/strategies.md

```diff
@@ -0,0 +1,134 @@
+# Strategies
+
+While a felt is a basis of all types in Cairo, its whole value space rarely matches what values are
+acceptable in tested code (for example, when using range checks builtin).
```

Comment:
```suggestion
Often in code we operate on some assumptions about the possible values of variables. 
For example we can perform greater than zero validation early in the code, and we can assume that variables are greater than zero in the subsequent code. 
Fuzzer allows to provide such assumptions to avoid testing against values which are not covered by the code. 
```

Reply:
I think it better explains strategies use case